### DISCUSSION TITLE
What about a Micropython port for ESP32-C6?

### DISCUSSION THREAD

**Discussion by bonne-soeur** [micropython/micropython#13461] [ESP32]
This is in fact a different discussion from the previously opened #11356
ESP32-C6 is a more recent RISC-V version of the ESP32 family. It is very close to the first risc-v item , the ESP32-C3. 
I read that only wifi and radio features and performances are different. Even though the chip ids are of course different . 
C3 has 5 whereas C6 has 13. 
Then , using esptool, even by forcing the flashing process of the Micropython firmware with the --force option, later on, when the MCU restarts on the Micropython firmware, it finds out the chip id incoherence and starts to loop (with the same 4 lines on the Thonny REPL window). 
Even if all the wifi features will not be supported until a true port is available, I think that a pseudo version using the C3 version but with the embedded C6 chip id inside the code at the right place may help us to start using the very nice and cheap boards available this winter, with the two usb-c interfaces. 
I know that the thread #11356 was already opened last april 2023, but now the situation is different as esptool perfectly supports (and detects) the esp32-C6.
So , dear George, Peter and Co. , would you give us some directions to start to experiment with those modules?

**Comment by ricksorensen** [comment]
See the ongoing work in this pull request:
#11869
